stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messageoptions/MessageOptions.kt

```diff
@@ -91,8 +91,11 @@ public fun defaultMessageOptionsState(
     currentUser: User?,
     isInThread: Boolean,
 ): List<MessageOptionItemState> {
+    val selectedMessageUserId = selectedMessage.user.id
+
     val isTextOnlyMessage = selectedMessage.text.isNotEmpty() && selectedMessage.attachments.isEmpty()
-    val isOwnMessage = selectedMessage.user.id == currentUser?.id
+    val isOwnMessage = selectedMessageUserId == currentUser?.id
+    val isUserMuted = currentUser?.mutes?.any { it.target.id == selectedMessageUserId } ?: true
```

Comment:
Maybe default it to `false`? Given that it's more common to have an unmuted user than a muted one?